src/components/FIcon.vue

```diff
@@ -132,4 +113,8 @@ const style = computed(
     '--ficon--stroke-width': genSize(props.strokeWidth),
   })
 );
+
+const name = toRef(props, 'name');
+const { markup } = useIcon('icons', name);
+const resolvedMarkup = computed(() => props.markup || markup.value);
```

Comment:
`name` is used once?
```suggestion
const { markup } = useIcon(toRef(() => props, 'name'));
const resolvedMarkup = computed(() => props.markup || markup.value);
```

Also, I used the recommended syntax for reactive prop getter: 
https://vuejs.org/api/reactivity-utilities.html#toref

Reply:
As stated in the documentation the getter syntax recommender for v3.3+ is: `toRef(() => props.name)` you mixed both ;-)

Reply:
BTW in fact I'm quite confused about this syntaxt, as they say:

> When toRef is used with component props, the usual restrictions around mutating the props still apply. Attempting to assign a new value to the ref is equivalent to trying to modify the prop directly and is not allowed.

so to my understanding this is exactly the same as doing `computed(() => props.name)` (except the type is `Readonly<Ref<...>>` instead of `ComputedRef<...>` 🤔

ChatGPT is a little but confused on that subject too ^^

Reply:
ChatGPT is not reliable on this topic I think (Vue 3.3 is quite new)
And sorry, I would say exactly like you say : `toRef(() => props.name)` (a computed like syntax btw)

Reply:
Yes indeed. My point is I don't get the difference between `computed(() => props.name)` and `toRef(() => props.name)` as they behave exactly the same